math-magic/src/components/CalculatorChild.js

```diff
@@ -0,0 +1,19 @@
+import './css/CalculatorChild.css';
+
+const Child = () => {
```

Comment:
- [OPTIONAL] Here the same as said with the component naming practice, you could use a better name for this component. Maybe something like `Buttons`